screeps-game-api/src/objects/mod.rs

```diff
@@ -194,6 +219,13 @@ pub unsafe trait RoomObjectProperties:
     }
 }
 
+/// Trait for all wrappers over Screeps JavaScript objects extending
+/// the `Structure` class.
+///
+/// # Contracts
```

Comment:
Since the `unsafe` here is carried from `RoomObjectProperties`, is it worth repeating?

Reply:
Oh! I forgot to edit these to actually not just be copy-pasted versions of the one above, which they are.

Thanks for catching that - let me know what you think of repetitiveness if they're actually reasonably different.